engines/joomla/nucleus/particles/contentarray.html.twig

```diff
@@ -21,7 +21,12 @@
     {% set categories = joomla.finder('category', category_options).published(1).language().limit(0).find() %}
 
     {# Content Finder #}
-    {% set article_finder = joomla.finder('content').category(categories).published(1).language() %}
+    {% if filter.articles %}
+        {% set article_options = filter.articles ? {id: [filter.articles|split(',')]} : {} %}
```

Comment:
Can you change

```twig
[filter.articles|split(',')]
```
to

```twig
[filter.articles|replace(' ', '')|split(',')]
```

This way is less prone to user mistakes when dealing with spaces and more future proof as now you can use `1,2, 5 , 6, 10` which will always be translated to `1,2,5,6,10`.